hstream-connector/HStream/Connector/Converter.hs

```diff
@@ -0,0 +1,25 @@
+module HStream.Connector.Converter
+where
```

Comment:
Please DO read this: https://docs.hstream.io/development/haskell-style/#export-lists

Reply:
I thought only one function will be fine... My fault.